### ISSUE TITLE
setup.sh is trying to wget from people.mozilla.org

### ISSUE THREAD

**Issue by glandium** [luser/breakpad-scrape-system-symbols#1]
And people.mozilla.org is gone. Also, wget is not installed by default on mac, while curl is.

**Comment by luser** [comment]
Yeah, this bitrotted. I don't have a good place to host prebuilt dump_syms binaries currently.
